tools/generate_requirements.py

```diff
@@ -26,6 +30,32 @@ def generate_requirement_file(name: str, req_list: list[str]) -> None:
     req_fname.write_text("\n".join(header + req_list) + "\n")
 
 
+def generate_environment_yml(req_sections: dict[str, list[str]]) -> None:
+    # Some packages in conda-forge have different names than they do on pypi
+    rename_idx = {
+        'build': 'python-build',
+        'kaleido': 'python-kaleido',
+        'sphinx_design': 'sphinx-design',
+    }
+    out = ["name: skimage-dev", "channels:", "  - conda-forge", "dependencies:"]
```

Comment:
sorry for the nit, but: I typically think of "out" as a buffer to which to write. So I'd rename this as "lines" or "yml_lines". (This comment brought to you by my staring for a few seconds thinking, "where are the strings going...?")

Reply:
Done